eks-distro-base/Dockerfile.minimal-base

```diff
@@ -60,13 +57,9 @@ COPY files/ $NEWROOT
 
 
 ################# BASE ########################
+FROM ${BUILT_BUILDER_IMAGE} as base-builder
 FROM ${BASE_IMAGE} as base
```

Comment:
I actually wouldnt mind a better name for this stage. We currently have 3:

builder
base
export

I think builder and export are fairly explanatory and base kind of makes sense, however the word `base` is also used to identify the most base minimal image.  It might be a nice change if we had a better name for this stage to avoid the ambiguity?

Reply:
How about something like `final` or `core`?